lesson2/sum mid and num.py

```diff
@@ -0,0 +1,18 @@
+if __name__ == '__main__':
+
+    summ = 0
+    i=float(input('Input number end press Enter: '))
+    num = []
```

Comment:
num > nums or numbers, because it is list

Reply:
Done